packages/office-ui-fabric-react/src/components/Label/Label.Props.ts

```diff
@@ -18,4 +18,9 @@ export interface ILabelProps extends React.HTMLProps<HTMLLabelElement> {
    * @defaultvalue false
    */
   required?: boolean;
+
+  /**
+  * Optional class for label.
+  */
+  className?: string;
```

Comment:
className is a standard HTML prop, you don't need to specify it on the props list. I.E. you can already pass through a className on this component.